test/functional/wallet_send.py

```diff
@@ -170,7 +170,7 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
                 else:
                     assert_greater_than(from_balance_before - from_balance, amount)
             else:
-                assert next((out for out in tx["vout"] if out["scriptPubKey"]["asm"] == "OP_RETURN 35"), None)
+                assert next((out for out in tx["vout"] if out["scriptPubKey"]["asm"] == "OP_RETURN 23"), None)
```

Comment:
This also seems confusing, particularly if it appears in something like `DUP SIZE 20 EQUALVERIFY HASH160 <x> EQUAL` -- whoops, that's actually saying the input should be 32 bytes, not 20 bytes.